apps/src/netsim/NetSimRouterLogModal.js

```diff
@@ -130,6 +137,18 @@ var NetSimRouterLogModal = module.exports = function (rootDiv, options) {
    */
   this.teacherView_ = false;
 
+  /**
+   * Pre-bound callback for use when rendering.
+   * @private {function}
+   */
+  this.boundSetRouterLogMode_ = this.setRouterLogMode_.bind(this);
```

Comment:
Is this considered good practice?  I guess it's analogous to what React does with its components.  It's clever, but I worry that it might make this class (which isn't React) harder to trace through down the road.